### ISSUE TITLE
Payment video not accessible for screen readers

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-design-system#904]
**WCAG Reference**:
- [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Issue ID**: DAC_Issue11
**URL**: https://design-system.service.gov.uk/patterns/payment-card-details/

## Screen Shot
The controls used in the video were not labelled for users that rely on audio feedback i.e. screen reader users and ZoomText with speech users.

## Screen reader comments
> “A member of the support team advised me that if the mouse was used to move to an image, video controls visually appeared on the page. I was not able to locate video controls when navigating in context with the arrow keys in JAWS and in a non-testing environment would have been unaware of the video. After investigation, with visual assistance, I found that with the tab key my focus visually reached the controls but the controls did not announce to me, with a paragraph of text reading instead for each control. I was therefore unable to interact with the video controls with JAWS as, once I selected the ‘Enter’ or ‘space’ keys, the page did not appear to change or update. I further found that when using the dialogue menus and other shortcut keys, for example ‘f’, the controls did not receive focus. With NVDA I was able to locate the ‘Play’ button while in context, however, once the button was selected, the controls vanished from the page and could not be located. I was able to interact with the video using VoiceOver for iPhone and Mac in the expected manner.”

## Low vision user comments
> “Whilst there was not an issue with accessing the multimedia itself, as when scrolling down the page on the left side a visible Play button was identifiable, there was no text reference to any multimedia above the video tile. Were a user to move down the page and scan across at the same time, they could possibly overlook the Play button. My expectation would have been to have found a Play symbol icon in the middle of the video image and a reference to it in the text above.”

## Solution
Ensure that all controls are clear and available to all users, although an aria label has been provided stating that no audio is present, users would still expect to be able to play the video.
The information that the video is referring to is present on the page and the video may not be required; consider using an image instead that contains a descriptive alt.

**Comment by nickcolley** [comment]
@selfthinker this might be useful for your video player research.

We are closing this and will investigate if this is an issue in Microsoft Edge and then report any bugs we find, but we don't believe we can fix this without building an entire video player component.

**Comment by nickcolley** [comment]
We think that given the content is explained outside of this video the video content is complementary.

We want to talk to the accessibility team to see if we should add this to our accessibility statement.

**Comment by selfthinker** [comment]
Although we agree that the native video player fails 1.3.1 and 4.1.2, we would say it is for the browser vendor to fix and not for us as it is clearly a browser bug.
If this can be fixed easily, we should fix it anyway (for moral, not legal reasons). But it looks like it would be more difficult in this case.

We should definitely report these issues to the relevant browser vendors. (Relevant bug reports might already exist.)

As soon as the 1.2.1 issue with the labelling is fixed, we don't think this needs to be in the Accessibility Statement.
But you can choose to add it into the statement voluntarily under a new section if you think it would be helpful for users to know about the issues with the native video player.

**Comment by nickcolley** [comment]
I've expanded the scope of our existing investigation card to cover more browsers and assistive technologies based on your recommendation: https://github.com/alphagov/govuk-design-system/issues/911

I've asked @amyhupe or @m-green to give their feedback on moving the label onto the page.

**Comment by amyhupe** [comment]
I'm happy with the suggestion. It would be good to make the wording slightly friendlier (apologies, I hadn't seen it before as the aria-label.

My suggestion would be to change it from:

"Visual demonstration of the card number validation user interaction described. This video has no audio."

To:

"This video shows the card number validation in practice. It doesn't have any audio."

**Comment by amyhupe** [comment]
We've improved the existing documentation based on @selfthinker's recommendations - ensuring that the wording is clearer and is exposed in the main body of the page.

We think this will help to make clear that the video is supplementary content and not required in order to understand the guidance. 

On this basis, we've decided not to include this in our statement, but will reconsider if we receive any feedback which suggests we should take a different approach.
